src/scss/settings/_buttons.scss

```diff
@@ -0,0 +1,79 @@
+// Sizes
+$buttonSizes: (
+  mini: (
+    fontSize: 12,
+    padding: (10 20)
+  ),
+
+  small: (
+    fontSize: 13,
+    padding: (11 22)
+  ),
+
+  regular: (
+    fontSize: 14,
+    padding: (16 32)
+  ),
+
+  big: (
+    fontSize: 16,
+    padding: (21 42)
+  )
+);
+
+// Types
+$buttonTypes: (
+  high-contrast: (
+    normal: (color: punch),
```

Comment:
@GeorgeJegon can we change `normal` for `default`? I think that is more understandable than `normal`.